lib/auth.js

```diff
@@ -151,9 +157,16 @@ Auth.authenticate = function authenticate(req, res, next) {
 
 
     function doAuth() {
-        req.ufds.authenticate(params.username, params.password, function (err, user) {
+        req.ufds.authenticate(username, params.password, function (err, user) {
             if (err) {
                 if (err.restCode === 'InvalidCredentials' || err.restCode === 'ResourceNotFound') {
+                    if (USERNAME_LOGIN_ATTEMPTS[username] > LOGIN_ATTEMPTS) {
+                        req.log.error({username: username},
+                            'Account temporarily locked after too many failed auth attempts via slave UFDS');
```

Comment:
Account is temporarily locked after too many failed auth attempts via slave UFDS